lgsm/functions/core_functions.sh

```diff
@@ -121,12 +121,15 @@ fn_fetch_function
 }
 
 command_postdetails.sh(){
-functionfile="${FUNCNAME}"
-tempffname=$functionfile
-fn_fetch_function
-functionfile="command_details.sh"
-fn_fetch_function
-functionfile=$tempffname
+    functionfile="${FUNCNAME}"
+    tempffname=$functionfile
+    # First, grab the command_postdetails.sh file
+    fn_fetch_function
+    # But then next, command_details.sh needs to also be pulled
+    # because command_postdetails.sh sources its functions -CedarLUG
+    functionfile="command_details.sh"
+    fn_fetch_function
+    functionfile=$tempffname
```

Comment:
$tempffname should be ${tempffname} all vars should have {} as it is the standard for LGSM